src/daemon/manager.py

```
@@ -151,3 +160,182 @@ def get_all_jobs_status() -> Dict[str, Any]:
+                                jobs_data.append({
+                                    "job_id": job_id,  # Use full job_id, not truncated display format
+                                    "seq_id": seq_id,
+                                    "last_run": last_run,
+                                    "next_run": next_run,
+                                    "status": run_state,
+                                    "total_execution_time": round(total_execution_time, 4),
+                                    "total_runs": total_runs,
+                                    "avg_execution_time": round(avg_execution_time, 4)
+                                })
+                                
+                            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)